rtcpeerconnection.js

```diff
@@ -294,15 +297,15 @@ module.exports = function(window, edgeVersion) {
   // Destroy ICE gatherer, ICE transport and DTLS transport.
   // Without triggering the callbacks.
   RTCPeerConnection.prototype._disposeIceAndDtlsTransports = function(
-    sdpMLineIndex) {
-    var iceGatherer = this._transceivers[sdpMLineIndex].iceGatherer;
+    transceiver) {
```

Comment:
unrelated to the PR: I think there needs to be a bunch of removeAllListeners() here.